### PR TITLE
Recent activity added to new home

### PR DESCRIPTION
**Pull request by BarbosaJackson**
added Recent Activity and a view to create logs about that.
Now created about flows, integrations and connect

### CODE REVIEW

File: connect/api/v1/recent_activity/views.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+class RecentActivityAPIView(views.APIView):
+    permission_classes = [ModuleHasPermission]
+
+    def post(self, request):
+        if request.data.get("flow_organization"):
+            project = Project.objects.filter(flow_organization=request.data.get("flow_organization"))
+        else:
+            project = Project.objects.filter(uuid=request.data.get("project_uuid"))
+        if len(project) > 0:
```
Comment (elitonzky) [inline connect/api/v1/recent_activity/views.py:20]: After if, for and among other logic blocks use a blank line
More details: https://peps.python.org/pep-0008/#blank-lines

**Review by elitonzky — CHANGES_REQUESTED** [review]
Small correction in: connect/api/v1/recent_activity/views.py

**Review by elitonzky — APPROVED** [review]

File: connect/common/models.py [hunk 2]
```diff
@@ -1808,3 +1808,66 @@ class TemplateProject(models.Model):
...
+                FLOW="edited-flow",
+                CHANNEL="edited-channel"
+            )
+        )
+        return actions[self.action][self.entity]
+
+    @property
+    def user_name(self):
+        # TODO: move to User model
+        return self.user.first_name + " " + self.user.last_name if self.user.first_name and self.user.last_name else self.user.email
```
Comment (elitonzky) [inline connect/common/models.py:1873]: I suggest this code below to make a line smaller:
 
        if self.user.first_name and self.user.last_name:
            return f"{self.user.first_name} {self.user.last_name}"

        return self.user.email

**Review by elitonzky — CHANGES_REQUESTED** [review]
code suggestion

File: connect/common/models.py (hunk 2 above)
Comment (BarbosaJackson) [reply connect/common/models.py:1873]: make sense

**Review by elitonzky — APPROVED** [review]

**Pull request merged**